test/parallel/sql/select.sql

```diff
@@ -0,0 +1,8 @@
+SELECT v AS v1111 FROM sift_base10k WHERE id = 1111 \gset
+SELECT v AS v2222 FROM sift_base10k WHERE id = 2222 \gset
+SELECT v AS v3333 FROM sift_base10k WHERE id = 3333 \gset
+SELECT v AS v4444 FROM sift_base10k WHERE id = 4444 \gset
+SELECT id FROM sift_base10k ORDER BY  v <-> :'v1111'  ASC LIMIT 1;
+SELECT id FROM sift_base10k ORDER BY  v <-> :'v2222'  ASC LIMIT 1;
```

Comment:
can we also have `EXPLAIN` case here with enable_seqscan=false , as after the merge of [this](https://github.com/lanterndata/lantern/pull/189) PR it won't be obvious that this was an index scan or not.